App/src/main/java/Shelter.Java

```diff
@@ -0,0 +1,95 @@
+public class Shelter{
+/*The shelter class contains contact information for homeless shelters, 
+and some validation logic for verifying phone numbers
+*/
+    private String name;
+    private String location;
+    private int phoneNum;
```

Comment:
Hey Jacinda good catch I completely forgot about that haha

Reply:
Good catch, Jacinda! I've actually often seen phone numbers represented as strings, because of all the different formats ((123) 456-7890 vs 123-456-7890, etc.). I really like that you're representing them as numbers instead, since that completely sidesteps the issue!